### PR TITLE
(for discussion) jasmine tests

### PR DESCRIPTION
**Pull request by c-monkey**
Thoughts? This front-end jasmine test calls calculateAndSave on the patient and hits the RESTful endpoint on the back end.

Are we liking this test approach, or am I off base?

### CODE REVIEW

File: spec/javascripts/helpers/measure_history_spec_loader.js.coffee [hunk 1]
```diff
@@ -32,7 +32,7 @@
 
     patients = new Thorax.Collections.Patients()
     try
-      patients.add(new Thorax.Models.Patient patientFixture) for patientFixture in getJSONFixture('records/' + path + '/patients.json')
+      patients.add(new Thorax.Models.Patient patientFixture, parse: true) for patientFixture in getJSONFixture('records/' + path + '/patients.json')
```
Comment (c-monkey) [inline spec/javascripts/helpers/measure_history_spec_loader.js.coffee:35]: By the way, this is the magic line that was most important, our helper method wasn't properly instantiating a Patient object fully

File: spec/javascripts/models/patient_spec.js.coffee [hunk 2]
```diff
@@ -92,3 +92,21 @@ describe 'Patient', ->
         success: =>
           expect(@patient.get('calc_results')).toBeDefined()
           done()
+
+describe 'Patient saving to backend', ->
+  # commented out for now because back end doesn't have this in the database, resulting in this error:
```
Comment (edeyoung) [inline spec/javascripts/models/patient_spec.js.coffee:97]: thoughts on how to address this?

**Comment by edeyoung** [comment]
@c-monkey what should we do about this? should we setup a meeting to discuss?

**Comment by c-monkey** [comment]
yeah, now that @alexanderelliott121 has completed the fixtures tools, I am sure we know more than we did before.

**Comment by c-monkey** [comment]
we can always revisit this someday, closing to remove any confusion.

**Pull request closed without merging**